.gitignore

```diff
@@ -2,6 +2,7 @@
 **/bin/
 !vendor/bin/premake/
 bin-int/
+*.log
```

Comment:
Maybe make a new group (aka a comment line above) for hazel files?
```.gitignore
# Hazel files
*.log
```

Reply:
done